web/ui/two-steps-page.svelte

```diff
@@ -5,16 +5,27 @@
 
   let prevTitle = document.title
 
+  let stepOneElement: HTMLDivElement
+
+  function loadNextFastHandler() {
+    stepOneElement.scrollTo({
+      top: 0,
+      behavior: 'smooth'
+    })
+  }
+
   onMount(() => {
     document.title = title + ' › ' + prevTitle
+    window.addEventListener('loadNextFast', loadNextFastHandler)
```

Comment:
`<TwoStepsPage>` is universal component used in many pages.

Having a hack just for a specific page is not long-term solution because a few month later it will be full of hacks for different pages.

What do you think of we will move this code to `web/pages/fast.svelte`?

You can export method from two-side-page and call it in fast page on fast posts store changes.

Reply:
@ai Should we provide a method for scrolling in the second column? And if so, is it worth making a more universal method, like this, or just two methods, for the first column and for the second??

```
 let stepFirstElement: HTMLDivElement
 let stepSecondElement: HTMLDivElement

  export function scrollStepToTop(
    stepName: 'stepFirstElement' | 'stepSecondElement'
  ) {
    function scrollToTop(element: HTMLDivElement) {
      element.scrollTo({
        top: 0,
        behavior: 'smooth'
      })
    }

    switch (stepName) {
      case 'stepFirstElement':
        scrollToTop(stepFirstElement)
        break

      case 'stepSecondElement':
        scrollToTop(stepSecondElement)
        break

      default:
        scrollToTop(stepFirstElement)
        break
    }
  }
```

Reply:
Yes, it will be nice to be able to scroll both side.

But note, that `stepFirstElement` string will not be minified by builder.

It is better to export 2 different functions: `scrollFirstToTop()` and `scrollSecondToTop()`.